core/src/main/scala/org/locationtech/rasterframes/util/DataFrameRenderers.scala

```diff
@@ -47,8 +49,9 @@ trait DataFrameRenderers {
               lit("\"></img>")
```

Comment:
Wondering if the line above this one
```
 base64(rf_render_png(rf_resample(resolved, 0.5), ColorRamps.Viridis))
```
should be changed to use `ColorRamps.greyscale(256)`, and then apply a color ramp in the python API for notebooks.

Reply:
I guess I am not sure about what we can do on the notebook / python side if the image is already b64 encoded in HTML... Some css stuff? idk

Reply:
Good point. Damn. Didn't think of that.